docs/_scripts/third_party_page/get_download_stats.py

```diff
@@ -37,18 +37,21 @@ def _get_weekly_downloads(packages: list[Package]) -> list[ResolvedPackage]:
     for package in packages:
         url = f"https://pypistats.org/api/packages/{package['name']}/overall"
 
-        response = requests.get(url)
-        response.raise_for_status()
-        data = response.json()
-
-        sorted_data = sorted(
-            data["data"],
-            key=lambda x: datetime.strptime(x["date"], "%Y-%m-%d"),
-            reverse=True,
-        )
-
-        # Sum the last 7 days of downloads
-        num_downloads = sum(entry["downloads"] for entry in sorted_data[:7])
+        try:
+            response = requests.get(url)
+            response.raise_for_status()
```

Comment:
why is this disabled?

Reply:
updated to be more explicit! was for handling recently pubbed packages